plugin.rb

```diff
@@ -86,7 +86,12 @@
       end
   end
 
-  add_to_serializer(:post, :preloaded_surveys, respect_plugin_enabled: false) do
+  add_to_serializer(
+    :post,
+    :preloaded_surveys,
+    respect_plugin_enabled: false,
+    include_condition: -> { false },
```

Comment:
so… it's never added? 👀

Reply:
Correct. I assume the default is `true`, and this disables it.

Reply:
I think it's purely used by the `#surveys` method.